api/turing/config/config.go

```diff
@@ -20,6 +20,9 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+// OpenapiBundleFile is the file name that is generated when the openapi bundle yaml file is generated.
+const OpenapiBundleFile = "openapi.bundle.yaml"
```

Comment:
I'm not sure if we should hardcode it in the go file. The name of the file is also specified in the make file: 
https://github.com/gojek/turing/blob/d8e0b2141f42d5883765e06d28c0116be2086d50/api/api/Makefile#L19
So with that, we will have two sources of truth of what filename should be used (and subsequently, we should update it in two places if we ever decide to rename this file). 
Is it possible to pass this a parameter?

Reply:
I'm quite conflicted on this. One one hand this value will never be changed so there is no need to make this configurable. I'm not sure how your solution will be implemented because there will always be two places this name will be hardcoded into.

What would your suggestion be?

Reply:
Changed it to an envvar